Sofa/Component/AnimationLoop/src/sofa/component/animationloop/init.cpp

```diff
@@ -54,4 +55,10 @@ void init()
     }
 }
 
+const char* getModuleComponentList()
+{
+    /// string containing the names of the classes provided by the plugin
+    static std::string classes = core::ObjectFactory::getInstance()->listClassesFromTarget(sofa_tostring(SOFA_TARGET));
```

Comment:
I think the constexpr var `MODULE_NAME` (which is defined for each module normally, and in its own namespace) would be better than to use the SOFA_TARGET preproc macro

Reply:
you're right. Done